src/common/helpers/responseHelper.ts

```diff
@@ -0,0 +1,70 @@
+import { Response } from "express";
+
+enum Status {
+    Error400 = 400,
+    Error401 = 401,
+    Error404 = 404,
+    Error422 = 422,
+    Error500 = 500,
+    Success200 = 200,
+}
+
+export function BadRequest(res: Response, reasonString: string): Response {
+    return JsonResponse(res, Status.Error400, "Bad request", reasonString);
+}
+
+export function Error401Response(res: Response, reasonString: string): Response {
+    return EndResponse(res, Status.Error401, reasonString);
+}
+
+export function NotFound(res: Response, reasonString: string): Response {
+    return JsonResponse(res, Status.Error404, "Not Found", reasonString);
+}
+
+export function MalformedRequest(res: Response, reasonString: string): Response {
+    return JsonResponse(res, Status.Error422, "Malformed request", reasonString);
+}
+
+export function Error422Response(res: Response, reasonString: string): Response {
+    return EndResponse(res, Status.Error422, reasonString);
+}
+
+export function Error500Response(res: Response, reasonString: string): Response {
+    return EndResponse(res, Status.Error500, `Internal server error: ${reasonString}`);
+}
+
+export function Success(res: Response): Response {
```

Comment:
See lines [36-70](https://github.com/UWPCommunity/uwpcommunity-backend/pull/10/commits/475a331d6bedd2b1efe0be32bf45de437b5cfa7b#diff-014843da1557fa9703dc8589d6a6a071R36-R70)

We won't be needing all these functions for different return methods. Using `res.send` should work when sending both JSON and strings ([and more](http://expressjs.com/en/4x/api.html#res.send)) and send the appropriate response headers. The only thing `res.send` can't do is send an empty response, for that you'll use `res.end`.